bundles/net.tourbook/src/net/tourbook/ui/views/TourBlogView.java

```diff
@@ -1465,6 +1471,7 @@ void updateUI() {
 
```

Comment:
Line 748: There should be space between the new tour summary and the tour title

![image](https://github.com/mytourbook/mytourbook/assets/1283445/76c29954-683b-4944-b456-9147992d6d8a)

Reply:
Fixed in this [PR](https://github.com/mytourbook/mytourbook/pull/1326)